README.md

```diff
@@ -79,3 +78,6 @@ The website is built using **Next.js**, a React-based framework for server-side
 3. In order to work on Sanity components, request to be added to Sanity.io as an administrator, and then populate the respective fields in the .env.local file.
```

Comment:
Supabase, not Sanity :-)

Reply:
LOL! I did not touch or write those but I will be happy to change them if want me to add them on this push.

```
2. Open local instance in browser: <http://localhost:3000>. Note: Pages populated by Sanity.io will return 404 until Sanity is properly configured.

3. In order to work on Sanity components, request to be added to Sanity.io as an administrator, and then populate the respective fields in the .env.local file.

4. In order to work on Airtable forms, request access to Airtable and create a personal access token```


Let me know what to update these too and I can make the change and add on PR.